### ISSUE TITLE
About 5-fold cross-validation technology

### ISSUE THREAD

**Issue by TRRtrr** [yxonic/DTransformer#5]
Hello, I'm sorry to bother you again. As I am not familiar with the 5-fold cross-validation technology in the KT field, I would like to ask you another question. Which of the following methods is commonly used for 5-fold cross-validation in the KT field?

Manually cut out 5 training sets, 5 validation sets, and 5 testing sets;

Manually cut out 5 training sets and 5 validation sets, and then share 1 test set;

Write a piece of code that can automatically extract 5 training sets, 5 validation sets, and 5 test sets randomly from the original dataset.

I look forward to your guidance very much, and I would greatly appreciate it. Thank you!
